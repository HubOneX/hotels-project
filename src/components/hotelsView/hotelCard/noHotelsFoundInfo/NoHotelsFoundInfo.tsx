import Typography from "@mui/material/Typography";
import React from "react";

const NoHotelsFoundInfo = () => {
  return (
    <Typography
      variant="h4"
      sx={{
        textAlign: "center",
        marginTop: "30px",
        lineHeight: "35px",
      }}
      data-testid="NoHotelsInfo"
    >
      Unfortunatelly, we couldn't find rooms meeting your requirements.
      <br />
      Please change your filters.
    </Typography>
  );
};

export default NoHotelsFoundInfo;
