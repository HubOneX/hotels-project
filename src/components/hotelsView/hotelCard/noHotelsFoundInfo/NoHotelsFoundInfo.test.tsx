import NoHotelsFoundInfo from "./NoHotelsFoundInfo";
import renderer from "react-test-renderer";

describe("NoHotelsFoundInfo", () => {
  it("should match the snapshot", () => {
    const container = renderer.create(<NoHotelsFoundInfo />);

    expect(container).toMatchSnapshot();
  });
});
