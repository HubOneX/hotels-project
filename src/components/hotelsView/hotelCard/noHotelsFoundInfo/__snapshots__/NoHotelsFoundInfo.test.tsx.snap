// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoHotelsFoundInfo should match the snapshot 1`] = `
<h4
  className="MuiTypography-root MuiTypography-h4 css-15u64ck-MuiTypography-root"
  data-testid="NoHotelsInfo"
>
  Unfortunatelly, we couldn't find rooms meeting your requirements.
  <br />
  Please change your filters.
</h4>
`;
